### ISSUE TITLE
Support changing parameters based on slope

### ISSUE THREAD

**Issue by idanarye** [idanarye/bevy-tnua#11]

**Comment by HugoPeters1024** [comment]
Would this include adjusting the rotation to make the model snap to the ground in these situations?

![image](https://github.com/idanarye/bevy-tnua/assets/22196124/c60e20bd-d761-4416-a9e9-dfa0b539d31e)

**Comment by idanarye** [comment]
No. This seems more like a job for the `desired_up` I've suggested in my comment to #40.
